src/lando/ui/jinja2/errorhandlers/revisions_404.html

```diff
@@ -1,21 +0,0 @@
-{% extends "partials/layout.html" %}
-{% block page_title %}Revision/Diff Not Available - Lando - Mozilla{% endblock %}
-
-{% block main %}
-<main class="ErrorPage container content">
-
-  <h1>Revision/Diff Not Available</h1>
-
-  <p>
-    The revision or diff you've requested does not exist or you do not have
-    permission to view it.
-  </p>
-  {% if user_is_authenticated and not user_has_phabricator_token %}
-    <p>
-      If you were trying to view a stack which includes a secure revision, you need to
```

Comment:
With this patch we remove the hint on the 404 page about adding the Phab API token. Could we add a `hint` parameter to the 404 page so we can add custom messaging to some 404's where appropriate? This message was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1808042 at Sylvestre's request.